browser/src/neovim/NeovimWindowManager.ts

```diff
@@ -172,12 +184,17 @@ export class NeovimWindowManager extends Utility.Disposable {
         currentWinId: number,
         context: EventContext,
     ): Promise<NeovimActiveWindowState> {
+        const positionCalls = [
+            ["nvim_call_function", ["line", ["w0"]]],
+            ["nvim_call_function", ["line", ["w$"]]],
+        ]
         // We query the top and bottom line positions again despite these being on the context
         // as the values from the `BufferUpdate` event can be incorrect
-        const [topLine, bottomLine] = await Promise.all([
-            this._neovimInstance.callFunction("line", ["w0"]),
-            this._neovimInstance.callFunction("line", ["w$"]),
-        ])
+        const [[topLine, bottomLine]] = await this._neovimInstance.request<[[number, number], any]>(
```

Comment:
The logic is the same I just swapped this to use the atomic calls api so it works the same way as the call below (also I'm not sure if the atomic calls might be more performant vs. promise.all)